app/dashboard/views.py

```diff
@@ -40,8 +44,53 @@
 confirm_time_minutes_target = 3
 
 
-def send_tip(request):
+@require_GET
+def github_callback(request):
```

Comment:
i haven't been horribly consistent about this myself... so take with a grain of salt.. but it could be worht moving all the github auth views into their own view file

Reply:
I agree. I originally added an app specifically for this, but figured it should conform to the rest of the project for the moment. 😂 I'll move it over to it's own app.